content/en/docs/concepts/windows/user-guide.md

```diff
@@ -3,38 +3,44 @@ reviewers:
 - jayunit100
 - jsturtevant
 - marosset
-title: Guide for scheduling Windows containers in Kubernetes
-content_type: concept
+title: Guide for Running Windows Containers in Kubernetes
+content_type: tutorial
 weight: 75
 ---
 
 <!-- overview -->
 
-Windows applications constitute a large portion of the services and applications that run in many organizations.
-This guide walks you through the steps to configure and deploy Windows containers in Kubernetes.
+This page provides a walkthrough for some steps you can follow to run
+Windows containers using Kubernetes.
+The page also highlights some Windows specific functionality within Kubernetes.
+
+It is important to note that creating and deploying services and workloads on Kubernetes
+behaves in much the same way for Linux and Windows containers.
+The [kubectl commands](/docs/reference/kubectl/) to interface with the cluster are identical.
+The examples in this page are provided to jumpstart your experience with Windows containers.
+
 
 <!-- body -->
 
 ## Objectives
 
-* Configure an example deployment to run Windows containers on the Windows node
-* Highlight Windows specific functionality in Kubernetes
+Configure an example deployment to run Windows containers on a Windows node.
+
+## {{% heading "prerequisites" %}}
 
-## Before you begin
+You should already have access to a Kubernetes cluster that includes a
+worker node running Windows Server.
 
-* Create a Kubernetes cluster that includes a control plane and a worker node running Windows Server
-* It is important to note that creating and deploying services and workloads on Kubernetes
-  behaves in much the same way for Linux and Windows containers.
-  [Kubectl commands](/docs/reference/kubectl/) to interface with the cluster are identical.
-  The example in the section below is provided to jumpstart your experience with Windows containers.
+The Windows node must be labelled as `kubernetes.io/os=windows`.
```

Comment:
This node label is added by the kubelet, should we include this line still?

https://kubernetes.io/docs/reference/labels-annotations-taints/#kubernetes-io-os
> Example: kubernetes.io/os: "linux"
>
>Used on: Node, Pod
>
> For nodes, the kubelet populates this with runtime.GOOS as defined by Go. This can be handy if you are mixing operating systems in your cluster (for example: mixing Linux and Windows nodes).

Reply:
```suggestion
```